articles/errors/deprecation-errors.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Deprecation Error Reference
+description: A listing of errors and descriptions relating to deprecations.
+toc: true
+---
+# Deprecation Error Reference
+
+## How to search logs for deprecation warnings
+
+There are two different ways to search for warning messages showing usage of deprecated features: The Auth0 Dashboard or the Auth0 Management API. Note that in either case, the [log retention period](/logs#how-long-is-log-file-data-available-) is governed by the subscription level of your account.
+
+### Search logs via the Dashboard
+
+In the [Auth0 Dashboard](/${manage_url}), if any deprecated features are being used, a "Deprecation Notice" message will show up in the **Logs** section of the dashboard. 
+
+::: note
+In order to not overwhelm the logs with repetetive messages, deprecation error messages will only be shown once per hour (the first time it occurs within that hour) rather than for each authentication transaction involving the deprecated feature.
+:::
+
+Navigate to the **Logs** screen in the Dashboard. Search for deprecation related messages by entering "type:depnote" in the query box.
+
+![Auth0 Dashboard - Logs](/media/articles/errors/libraries/dashboard-logs.png)
+
+A list of deprecation related warning messages from your logs will be shown, if any exist.
+
+The **Description** field provides information on the particular deprecated feature used. Clicking on the link in the **Event** column for each item will show additional information such as the client id which identifies the client application using the deprecated feature.
+
+### Search logs via the Management API
+
+Customers can also use the Management API to search through logs for such messages by looking for  "Type" = "depnote".
+
+To check your logs using the Management API, go to the [Management API](/api/management/v2).
+
+If you have not already done so, [get and set up your API token](/api/management/v2/tokens#get-a-token-manually) in the API explorer.
+
+![Management API - Token Setup](/media/articles/errors/libraries/management-api-set-token.png)
+
+On the left, navigate to **Logs > Search log events** and then scroll down to **Parameters**.
+
+![Management API - Logs](/media/articles/errors/libraries/management-api-logs.png)
+
+In the "q" field enter: `type:"depnote"`
+
+Click on the **TRY** button. If successful, you should see a screen similar to the one below.
+
+![Management API - Logs - Results](/media/articles/errors/libraries/management-api-logs-results.png)
+
+* The results will match one of the messages + descriptions below.
+* The "Client ID" field will indicate which application (client) on your tenant is using the deprecated feature.
```

Comment:
"Client ID" -> **Client ID** 
btw which is this field? is it part of the results if you have deprecation warnings? It left me wondering since I didn't see it anywhere